src/pages/SignIn.js

```diff
@@ -20,12 +19,14 @@ const styles = {
 }
 
 const SignIn = () => {
+  const brand = getEnabledBrand()
+
   return h(HeroWrapper, { showMenu: false, showDocLink: true }, [
     div({ style: { maxWidth: 600 } }, [
       div({ style: { fontSize: 36, color: colors.dark(0.6) } }, ['New User?']),
-      div({ style: { fontSize: 36, marginBottom: '2rem' } }, [`${getAppName({ capitalizeThe: true })} requires a Google Account.`]),
+      div({ style: { fontSize: 36, marginBottom: '2rem' } }, [`${brand.name} requires a Google Account.`]),
       div({ style: { fontSize: 16, lineHeight: 1.5, marginBottom: '2rem' } }, [
-        `${getAppName({ capitalizeThe: true })} uses your Google account. Once you have signed in and completed the user profile registration step, you can start using ${getAppName()}.`
+        `Once you have signed in and completed the user profile registration step, you can start using ${brand.signInName}.`
```

Comment:
`brand.signInName` is used both for a user-facing label here and also to construct return URLs for Firecloud. Should those two things use separate configuration values? Are they guaranteed to always use the same values?

Reply:
Changed in [a8e8598](https://github.com/DataBiosphere/terra-ui/pull/3257/commits/a8e8598f0c8066356145bf35c6acf3ddcd4dbae6).